docs/integrations/sources/hardcoded-records.md

```diff
@@ -0,0 +1,35 @@
+# Hardcoded Records
+
+## Sync overview
+
+The Sample Data (Hardcoded Records) source outputs sample data using the hardcoded .
+
+### Output schema
+
+This source will generate an "e-commerce-like" dataset with users, products, and purchases. Here's
+what is produced at a Postgres destination connected to this source:
```

Comment:
Don't forget the schema (but ultimately I think it's fine to omit tbh).

Reply:
updated